style.css

```diff
@@ -597,11 +704,74 @@ label {
     object-position: center;
   }
 
+  .pData {
+    position: absolute;
+    visibility: hidden;
+    zoom: 0.9;
+    opacity: 0%;
+    transition: all 180ms cubic-bezier(0.225, 0.595, 0.88, 0.39);
+    transition-timing-function: cubic-bezier(0.225, 0.595, 0.88, 0.39);
+  }
+
   .card2 .pData,
   .card6 .pData {
     zoom: 1.5;
   }
 
+  .card4 {
+    grid-column: 2 / 4;
+    grid-row: 2 / 4;
+    background-image: url(images/project/image-geometry_2.png);
+    background-repeat: no-repeat;
+    background-position: bottom;
+    background-size: contain;
+  }
+
+  .card8 {
+    grid-column: 1 / 3;
+    grid-row: 6;
+    background-image:
+      url(images/project/image-geometry_4.png),
+      url(images/project/image-geometry_5.png);
+    background-repeat: no-repeat;
+    background-position: top right, top left;
+    zoom: 110%;
+    height: 100%;
+  }
+
+  .card9 {
+    grid-column: 3 / 5;
+    grid-row: 6;
+  }
+
+  .card9 img {
+    width: 70%;
+  }
+
+  .card9 .pData {
+    width: 70%;
+  }
+
+  /* project section descktop */
+
+  .Projects {
+    display: flex;
+    max-width: 1280px;
+    margin: 15vh auto;
+  }
+
+  .card-back2,
+  .card-back3,
+  .card-back4,
+  .form-name-message p {
+    display: block;
+  }
+
+  .card2 .pData,
```

Comment:
- [ ] On the desktop as you have implemented the popup details for the `see this project` button please Kindly implemet the details popup on the first button `see this project`.This will make your website look completely professional like the figma template and user friendly, you can see the screenshot below .
  
- [ ]  
![ggggggggg](https://github.com/Bleu-Yves-Sopoude/mb-portfolio/assets/92458236/7402bc37-4a06-451b-a97e-91550cbcae3f)